README.md

```diff
@@ -1,3 +1,5 @@
+[![CircleCI](https://circleci.com/gh/datopian/ckanext-geodatagov.svg?style=svg)](https://circleci.com/gh/datopian/ckanext-geodatagov)
```

Comment:
Should be GSA repo.

Reply:
fixed [here](https://github.com/datopian/ckanext-geodatagov/commit/41307320db41d9285fa4f8f56b1cf70d3d356546)